shopfloor_mobile_base/static/src/components/detail/detail_card.esm.js

```diff
@@ -15,7 +15,7 @@ Vue.component("item-detail-card", {
             <v-card-title v-if="!opts.no_title" :class="opts.title_class">
                 <slot name="title">
                     <v-icon v-if="opts.title_icon" v-text="opts.title_icon" class="mr-2" />
-                    <span v-text="_.result(record, opts.key_title)" />
+                    <span v-text="_.result(record, opts.key_title, opts.title_default || '')" />
```

Comment:
What is the purpose of this option? Force a title or just use a fallback? What is the context?

Reply:
It allows to set a default value for the title if the field fetched on the record returned a falsy value.
It is used https://github.com/OCA/stock-logistics-shopfloor/pull/44/files#diff-7e58c2adffee7ac71f5b51e34ef54e5b4b92e59eb2d72797ed88dd5ce07f2941R466